sos/src/sos_cmd.c

```diff
@@ -606,12 +607,18 @@ void json_row(FILE *outp, sos_schema_t schema, sos_obj_t obj)
 		} else {
 			col_str = malloc(col_len);
 		}
-		if (sos_attr_is_array(attr) && sos_attr_type(attr) != SOS_TYPE_CHAR_ARRAY) {
-			fprintf(outp, "\"%s\" : [%s]", col->name,
-				sos_obj_attr_to_str(obj, attr, col_str, col_len));
+		s = sos_obj_attr_to_str(obj, attr, col_str, col_len);
+		if (!s) {
+			fprintf(outp, "\"%s\" : null", col->name);
```

Comment:
Isn't this going to make alll types strings? What if it's a float or an int? So instead of getting:

"job_id" : 1324,

we'll get

"job_id : "1324"

or am I missing something?

Reply:
Yes, [line 620 in the change](https://github.com/ovis-hpc/sos/pull/57/files#diff-1e29f017077b212fcb739b9fa48810fc1ddae4a70f49dc40123d1c361b256f80R620) will print all non-array types as strings. I followed the design before making the change. (See [line 613 on the left](https://github.com/ovis-hpc/sos/pull/57/files#diff-1e29f017077b212fcb739b9fa48810fc1ddae4a70f49dc40123d1c361b256f80L613). I will change it so that sos_cmd will print all integer and float without the double quote.

Reply:
I changed the code so that it doesn't print numbers as strings.